gql/Fragments.graphql

```diff
@@ -2,7 +2,10 @@ fragment reactionFields on Reactable {
   viewerCanReact
   reactionGroups {
     viewerHasReacted
-    users {
+    users(first: 10) {
```

Comment:
10 seems a bit high, no? I like your example using up to 3 before showing "n others".